pages/linux/expr.md

```diff
@@ -0,0 +1,23 @@
+# expr
+
+> Evaluate expressions.
+
+- Get string length:
+
+`expr length {{string}}`
+
+- Evaluate logical and math expression (special symbols should be escaped):
+
+`expr {{ARG1}} {{operator}} {{ARG2}}`
```

Comment:
You should give at least one or two examples of possible operators. If there are few (say, about 4), maybe inculde them all inside the `{{}}` separated by a pipe (or some other character? /cc @ostera @rubenvereecken)